src/ansys/dpf/gate/misc.py

```diff
@@ -1,11 +1,33 @@
 DEFAULT_FILE_CHUNK_SIZE = None
 COMMON_PROGRESS_BAR = None
-_CLIENT_CONFIG = {}
+
```

Comment:
@cbellot000 do we have an equivalent PR on pygate?

Reply:
yes